tests/test_version.py

```diff
@@ -0,0 +1,16 @@
+import os
+import re
+import unittest
+
+
+class TestVersion(unittest.TestCase):
```

Comment:
Do we really need these tests? 
They'll always run in `python -m unittest` 
It's fine if'll get an error in CI before publishing

Reply:
removed and added only to CI